.github/workflows/_hugo.yml

```
@@ -0,0 +1,55 @@
+name: Build Hugo Site
+
+on:
+  workflow_call:
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    outputs:
+      page_url: ${{ steps.some_step.outputs.page_url }}
```

actionlint

property "some_step" is not defined in object type {pages: {conclusion: string; outcome: string; outputs: {base_path: string; base_url: string; host: string; origin: string}}}

(expression)